src/utils/BalancedGrowthCommunityModel.jl

```diff
@@ -0,0 +1,22 @@
+"""
+$(TYPEDSIGNATURES)
+
+Extract the solution of a specific `community_member` from `opt_model`, which is
+a solved optimization model built from the `community_model`. Removes the
+`community_member` prefix in the string ids of the returned dictionary.
+"""
+get_solution(community_member::CommunityMember, opt_model, community_model::BalancedGrowthCommunityModel) = is_solved(opt_model) ? Dict(
+    string(last(split(rid, community_member.id*"#"))) => val for (rid, val) in zip(reactions(community_model), reaction_variables(community_model)' * value.(opt_model[:x])) if startswith(rid, community_member.id*"#")
+) : nothing
+
+
+"""
+$(TYPEDSIGNATURES)
+
+Extract the environmental exchanges from `opt_model`, which is a solved
+optimization model built from the `community_model`.
+"""
+get_environmental_exchanges(opt_model, community_model::BalancedGrowthCommunityModel) = is_solved(opt_model) ? Dict(
```

Comment:
consider keeping the arguments in a consistent order, preferably sorted by "logical size" ( model ≥ optmodel ≥ piece of model ).

Other than that I'm okay with this.

Reply:
cool, will change!